hmpps-mysql-tools/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:20.04
```

Comment:
Using ubuntu (instead of alpine) as installing locales with alpine seems to be a bit difficult.